docs/source/configuration/metrics.mdx

```diff
@@ -72,3 +73,22 @@ telemetry:
       # Optional timeout in humatime form
       timeout: 2s
 ```
+
+## Add custom labels
+
+If you need to add labels on your generated metrics, for example add a custom labels coming from a specific header in the original HTTP request, or insert custom labels on every metrics.
+
+```yaml title="router.yaml"
+telemetry:
+  metrics:
+    common:
+      additionnal_labels:
```

Comment:
In otel metrics I think these are called attributes rather than labels. If this is the case let's update the config format.

Related, maybe from_headers rather than propagate_headers. The latter implies moving from one header to another header.

Reply:
If this could wait for after the next release then we could have time to discuss.